src/queries/daily-rum.sql

```diff
@@ -0,0 +1,22 @@
+--- description: demo rum data
+--- Authorization: none
+--- Access-Control-Allow-Origin: *
+--- offset: 0
+--- repo: express-website
+
+SELECT
+  COUNT(*) AS URLHITS,
+  AVG(LCP) AS AVGLCP,
+  AVG(FID) AS AVGFID,
+  AVG(CLS) AS AVGCLS,
+  EXTRACT(MONTH FROM TIMESTAMP_MILLIS(CAST(TIME AS int64))) AS MONTH,
+  EXTRACT(DAY FROM TIMESTAMP_MILLIS(CAST(TIME AS int64))) AS DAY,
+  EXTRACT(YEAR FROM TIMESTAMP_MILLIS(CAST(TIME AS int64))) AS YEAR
+FROM `helix-225321.helix_rum.rum20*`
```

Comment:
This should never have been merged. You are using the wrong (soon to be deprecated) data source. I'll remove this in #831